Sources/Prometheus/RELEASE.md

```diff
@@ -0,0 +1,50 @@
+# Releasing 
+
+## Release schedule
+
+Releases of this library are performed on a when-needed basis. 
+
+Generally one of the maintainers listed in MAINTAINERS.md will perform a release.
+
+If you would like to request a release since a change or fix has been waiting for a release for some time,
+you may open up an issue on github, requesting a release.
+
+## Versioning 
+
+As usual with Swift packages, we use [Semantic Versioning](https://semver.org/).
+
+## Make a release
+
+Development and releases are made from the `main` branch directly, therefore the branch should remain in releasable state at any time.
+
+Making a release is simple and boils down to making a git tag and pushing it to the upstream repository.
+
+Tags should follow the simple `MAJOR.MINOR.PATCH` format, without any prefix (e.g. do not include the `v` prefix).
+
+Step 1) Make a release tag.
+
+```bash
+tag="MAJOR.MINOR.PATCH"
+git tag -s "${tag}" -m "${tag}"
+git push origin "${tag}"
+```
```

Comment:
This is not really needed. When using the Github release UI it will automatically create the tag for you

Reply:
Does it sign the tags actually? 🤔

Reply:
Fine to keep this.